azure-spring-boot-starters/azure-mediaservices-spring-boot-starter/README.md

```diff
@@ -18,8 +18,10 @@ If you are using Maven, add the following dependency.
 Open `application.properties` file and add below properties with your Azure Media Services credentials.
 
 ```
-azure.mediaservices.account-name=put-your-media-services-account-name-here
-azure.mediaservices.account-key=put-your-media-services-account-key-here
+azure.mediaservices.tenant=put-your-media-service-azure-ad-tenant-domain-here
+azure.mediaservices.client-id=put-your-azure-ad-client-id-here
+azure.mediaservices.client-secret=put-your-azure-ad-client-secret-here
+azure.mediaservices.rest-api-endpoint=put-your-media-service-rest-api-endpoint-here
```

Comment:
How about endpoint?

Reply:
Prop name is borrowed from media service doc, as multiple kinds of endpoints exist, using rest-api-endpoint might be more accurate, or  remove the `api` and use `rest-endpoint`?